src/cogs/interactions/slash/anon.py

```diff
@@ -0,0 +1,565 @@
+import discord
+import datetime
+from bot import DiscordBot
+from discord import app_commands
+from discord.ext import commands, tasks
+from utils import general as ug
+from typing import Optional
+
+
+class SlashAnon(commands.Cog):
+    def __init__(self, client: DiscordBot):
+        self.client = client
+        self.anon_cache = {}
```

Comment:
Enhancement: it'll help if the anon_cache is a map with key as the message id and value as whatever you need it to have (timestamp, user id, etc). easier for lookup for bans, cache clear, etc